code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -343,6 +344,23 @@
 
 	return message
 
+/datum/outfit/traitor
+	name = "Traitor (Preview only)"
+
+	uniform = /obj/item/clothing/under/color/grey
+	suit = /obj/item/clothing/suit/hooded/ablative
+	gloves = /obj/item/clothing/gloves/color/yellow
+	mask = /obj/item/clothing/mask/gas
+	l_hand = /obj/item/melee/energy/sword
+	r_hand = /obj/item/gun/energy/kinetic_accelerator/crossbow
+
+/datum/outfit/traitor/post_equip(mob/living/carbon/human/H, visualsOnly)
+	var/obj/item/melee/energy/sword/sword = locate() in H.held_items
+	sword.icon_state = "swordred"
+	sword.worn_icon_state = "swordred"
```

Comment:
I don't think these icon_states exist anymore, renamed to `e_sword_on_red`

Reply:
![image](https://user-images.githubusercontent.com/35135081/132942098-6bb1a17f-6bbb-4617-9358-37bf06b6a855.png)
🎉